dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task201008LoadAppsSecretsTest.java

```diff
@@ -0,0 +1,180 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.datagen.AppDescriptorDataGen;
+import com.dotcms.datagen.SiteDataGen;
+import com.dotcms.datagen.TestUserUtils;
+import com.dotcms.security.apps.AppDescriptor;
+import com.dotcms.security.apps.AppDescriptorHelper;
+import com.dotcms.security.apps.AppSecrets;
+import com.dotcms.security.apps.AppsAPI;
+import com.dotcms.security.apps.AppsCache;
+import com.dotcms.security.apps.AppsUtil;
+import com.dotcms.security.apps.SecretsKeyStoreHelper;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.CacheLocator;
+import com.dotmarketing.exception.AlreadyExistException;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.contentlet.business.HostAPI;
+import com.google.common.collect.ImmutableSet;
+import com.liferay.portal.model.User;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.security.Key;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task201008LoadAppsSecretsTest {
+
+    static AppsAPI api;
+    static User admin;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)